### PR TITLE
Tips and troubleshooting for installing packages

### PR DESCRIPTION
**Pull request by davidread**
I've copied and improved some of the tips from our wiki and what I've gathered from support requests.

I'm no expert on all this - I'd appreciate a proper review by knowledgable R people.

### CODE REVIEW

File: source/documentation/tools.md [hunk 1]
```diff
@@ -41,6 +41,102 @@ For Python:
...
+```
+
+There are a few options to avoid this:
+
+Solution 1: AP may have a newer version of RStudio tool which might have the version of R needed. To upgrade, see: [Managing your analytical tools](managing-your-analytical-tools)
+
+Solution 2: Use [conda](#conda) - it's recommended for use with Analytical Platform in general. It works out which version is compatible with your R version (make sure you run this in the Terminal):
+
+```bash
+conda install r-test2vec
```
Comment (samtazzyman) [inline source/documentation/tools.md:77]: should be `text2vec` not `test2vec`

File: source/documentation/tools.md [hunk 2]
```diff
@@ -41,6 +41,102 @@ For Python:
...
+```
+
+Solution 3: Specify a version that is compatible with your R version. e.g. at https://www.rdocumentation.org/packages/text2vec look at the "depends" field for the R version it requires. Change the version (drop-down at the top) to go back to see how it changes for older releases. You can see that text2vec 6.0 requires R (>= 3.6.0), but text2vec 5.1 requires only R (>= 3.2.0).
+
+```R
+devtools::install_version('text2vec', version='0.5.1')
```
Comment (samtazzyman) [inline source/documentation/tools.md:83]: do we assume they've got/know how to get `devtools`?

File: source/documentation/tools.md [hunk 3]
```diff
@@ -41,6 +41,102 @@ For Python:
...
+
+This happens when a package on conda is marked as _broken_. r-pillar seems to suffer this frequently.
+
+To fix this:
+
+1. Remove `r-pillar` (or the offending package) from environment.yml. `r-pillar` is provided by the base conda environment and chances are that the user doesn't need it in their app, so it can be safely removed.
+
+2. Update the version of `r-pillar` to the [latest one](https://anaconda.org/conda-forge/r-pillar/files) on conda-forge 
+
+3. Update the base environment. The above two fixes are quick but don't solve the underlying problem that one of our dependencies in the base environment has been marked as broken. The ultimate fix is to update the [base packages](https://github.com/ministryofjustice/analytics-platform-rstudio/blob/master/files/conda_packages).
```
Comment (samtazzyman) [inline source/documentation/tools.md:138]: Is this "ultimate fix" for us to do, or for the user to do? If it's for us I'm not sure of the utility of including it here in the user guidance.

Comment (davidread) [reply source/documentation/tools.md]: Agreed, this is for AP team, so I've removed this option.

File: source/documentation/tools.md (hunk 2 above)
Comment (davidread) [reply source/documentation/tools.md:83]: It's on the list of packages we install in conda environment by default, so I think that's good for the vast majority

File: source/documentation/tools.md (hunk 1 above)
Comment (davidread) [reply source/documentation/tools.md]: 👍 Fixed

**Pull request merged**